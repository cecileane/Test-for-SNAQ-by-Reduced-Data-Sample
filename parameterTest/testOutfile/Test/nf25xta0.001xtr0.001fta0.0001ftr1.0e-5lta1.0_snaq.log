optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 1.0.
rootname for files: nf25xta0.001xtr0.001fta0.0001ftr1.0e-5lta1.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 10577
seed: 10577 for run 1
Sat Oct  1 17:32:18 2016

 BEGIN SNaQ for run 1, seed 10577 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 37 (failures: 25) with -loglik=126.74251 and ht_min=[0.2856,1.51213,0.51916,10.0,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 8		 8		 0	 |	 0		 0		 --
mvtarget	 10		 10		 1	 |	 0		 0		 --
chdir		 4		 4		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 13		 11		 2	 |	 0		 0		 --
Total		 37		 35		 5	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981976573
((5,(3,(4,(2,(1)#H7:::0.7143969204463817):1.5121330055957978):0.4894187015971525):0.5191606156078314):10.0,6,#H7:::0.2856030795536183);
---------------------
seed: 35547 for run 2
Sat Oct  1 17:32:22 2016

 BEGIN SNaQ for run 2, seed 35547 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 48 (failures: 25) with -loglik=126.74251 and ht_min=[0.2856,0.51916,10.0,1.51213,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 4		 4		 1	 |	 NA 		 NA 		 NA 
mvorigin	 9		 9		 1	 |	 0		 0		 --
mvtarget	 8		 8		 0	 |	 0		 0		 --
chdir		 9		 9		 3	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 18		 16		 3	 |	 1		 1		 --
Total		 48		 46		 8	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.2	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975321
(((4,(2,(1)#H7:::0.7143969610988488):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7:::0.28560303890115124):10.0,5);
---------------------

Sat Oct  1 17:32:25 2016
MaxNet is (((4,(2,(1)#H7:::0.7143969610988488):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7:::0.28560303890115124):10.0,5); 
with -loglik 126.74250981975321
