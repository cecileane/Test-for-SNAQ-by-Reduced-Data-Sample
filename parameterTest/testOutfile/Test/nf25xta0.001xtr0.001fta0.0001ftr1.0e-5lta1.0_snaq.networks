(((4,(2,(1)#H7:::0.7143969610988488):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7:::0.28560303890115124):10.0,5);, with -loglik 126.74250981975321 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((4,(2,(#H7:::0.29268991355420754,1):7.474230791969216):0.7121362508697179):0.47011076385357425,3):1.4419932260911077,(6)#H7:::0.7073100864457924,5);, with -loglik 128.82856135324585
(((4)#H7:::0.5923617163581254,3):0.05424749885596614,(6,((2,#H7:::0.4076382836418746):2.5828963854896383,1):1.279085172302787):0.2176349417906912,5);, with -loglik 173.19195575555977
(((4,#H7:::0.4300951842350773):1.7011881417241654,3):0.03571847428049011,(6,((2)#H7:::0.5699048157649227,1):9.322565301615462):0.15827849199598065,5);, with -loglik 191.84642741011254
(#H7:::0.4066979484361548,(6,((2,(4,(3)#H7:::0.5933020515638452):0.07577364617791643):0.0944599217756347,1):1.0021070096098397):4.137015341484447,5);, with -loglik 214.96637351732298
(4,(2,((6,#H7:::0.4909594790006751):8.857867697607366,1):0.043943111781367714):0.20917164672009841,((5)#H7:::0.509040520999325,3):1.1481419537950301);, with -loglik 230.99674126908292
