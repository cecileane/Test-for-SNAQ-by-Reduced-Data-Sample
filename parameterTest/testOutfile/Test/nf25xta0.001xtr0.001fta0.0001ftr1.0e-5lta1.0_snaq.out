(((4,(2,(1)#H7:::0.7143969610988488):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7:::0.28560303890115124):10.0,5); -Ploglik = 126.74250981975321
 Dendroscope: (((4,(2,(1)#H7):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7):10.0,5);
 Elapsed time: 6.596445798 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((4,(2,(1)#H7:::0.7143969610988488):1.5121332542336658):0.4894189320993494,3):0.5191606100345654,(6,#H7:::0.28560303890115124):10.0,5);, with -loglik 126.74250981975321
 ((5,(3,(4,(2,(1)#H7:::0.7143969204463817):1.5121330055957978):0.4894187015971525):0.5191606156078314):10.0,6,#H7:::0.2856030795536183);, with -loglik 126.74250981976573
-------
